test/Utils.test.ts

```diff
@@ -4,8 +4,8 @@ import {Utils} from "../src/Utils";
 describe('Utils ->', () => {
     describe('generateClientLibsUrl', () => {
         it('generateClientLibsUrl', () => {
```

Comment:
Please provide human-readable strings. ~  _it('should generate the expected number of client library URLs..._

Reply:
Done.